DemoApp/Sources/Views/Reactions/DemoReactionsOverlayView.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+import StreamChatSwiftUI
+import SwiftUI
+
+struct DemoReactionsOverlayView<Factory: ViewFactory>: View {
+
+    var factory: Factory
+    var channel: ChatChannel
+    var currentSnapshot: UIImage
+    var messageDisplayInfo: MessageDisplayInfo
+    var onBackgroundTap: () -> Void
+    var onActionExecuted: (MessageActionInfo) -> Void
+
+    var body: some View {
+        ReactionsOverlayView(
+            factory: factory,
+            channel: channel,
+            currentSnapshot: currentSnapshot,
+            messageDisplayInfo: messageDisplayInfo,
+            onBackgroundTap: onBackgroundTap,
+            onActionExecuted: onActionExecuted
+        )
+        .offset(y: -29)
```

Comment:
-29?

Reply:
To avoid covering the connection status view

Reply:
if possible, it should be a multiple of 8, or at least even. But leave it up to you.